Documentation/Development.md

```diff
@@ -80,14 +80,14 @@ using a [snapshot](https://swift.org/download/#releases) from swift.org.
 	$ /path/to/swiftpm/.build/x86_64-apple-macosx/debug/swift-build
 	```
 
-5. Testing the Swift Package Manager.
+5. Test the Swift Package Manager.
 
 	```sh
 	$ Utilities/bootstrap test --test-parallel
 	```
 	Use this command to run the tests. All tests must pass before a patch can be accepted.
 
-## Self-hosting
+## Self Hosting
```

Comment:
Can we change this to a verb phrase to be a little more consistent in our headings. How about `Self Hosting a Swift Package` or `Hosting a Swift Package on Your Own Server`?

Reply:
Ageed, and this is part of a larger update that changes the headings to match this style.